### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.4.7][Eclipse][Wipro]:Keyboard focus is not visible for pane present in "Sign in with a username and password instead" dialog 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1526]
**Repro Steps:**

1. Open and Start Eclipse
2. Tab to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Interactive" radio button.
4. Tab to and select "Sign in" button.
5.  In" Azure Login" Enter the "email or phone number" in the edit box and tab to Next button then hit enter.
6.  Tab to and select "Sign in with a user name and password instead" link.
7.  Tab up to "Help and support" link present in bottom of the page then press tab Observe the keyboard behavior.

**Expected Result:**
Keyboard focus should be visible for  pane present in  Azure Login Dialog

**Actual Result:**
Keyboard focus is not visible for  pane present in Azure Login Pane

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![keyboard focus is not visible for pane in azure login dialog](https://user-images.githubusercontent.com/22956130/36191749-aeac69e8-1184-11e8-9b7f-52a211d96bd2.jpg)

**Comment by Eskibear** [comment]
duplicated with #1525

**Comment by WiproPSAccTest** [comment]
Verified in V3.6.0.
Issue still exists .
Keyboard focus is not visible for pane present in Azure Login Pane

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749969
